src/TimelineID/TimelineId.tsx

```diff
@@ -1,12 +1,131 @@
+import { useParams, Link } from "react-router-dom";
+import { VerticalTimeline, VerticalTimelineElement } from 'react-vertical-timeline-component';
+import 'react-vertical-timeline-component/style.min.css';
 
-
+interface Rover {
+    name: string;
+    information: string;
+    informationSectionOne: string;
+    imageUrl: string;
+}
 export const TimelineId: FunctionComponent = () => {
-    const id = "";
+
+    const { roverName }: any = useParams();
+    const rover: Rover = getRover(roverName)!;
+
     return (
-        <div className={styles.timelineID}>
-            <h3>The current Timeline Id is: {id}</h3>
-        </div>
-    );
-}
\ No newline at end of file
+        <section className={styles.BackgroundColor}>
+            <VerticalTimeline>
+                <VerticalTimelineElement
+                    className="vertical-timeline-element--work"
+                    contentStyle={{ background: '#631111', color: '#fff' }}
+                    contentArrowStyle={{ borderRight: '7px solid  #631111' }}
+                    date="2011 - present"
+                    iconStyle={{ background: '#631111', color: '#631111' }}
+                >
+                    <h3 className={styles.TimeLineHeading}>{rover.name}</h3>
+                    <img className={styles.RoverImage} src={rover.imageUrl} alt="rover" />
+                    <p className={styles.RoverInformation}>
+                        {rover.information}
+                    </p>
+                </VerticalTimelineElement>
+
+                <VerticalTimelineElement
+                    className="vertical-timeline-element--work"
+                    date="2010 - 2011"
+                    contentStyle={{ background: '#631111', color: '#fff' }}
+                    contentArrowStyle={{ borderRight: '7px solid  #631111' }}
+                    iconStyle={{ background: '#631111', color: '#fff' }}
+                >
+                    <h3 className={styles.TimeLineHeading}>{rover.name}</h3>
+                    <img className={styles.RoverImage} src={rover.imageUrl} alt="rover" />
+                    <p className={styles.RoverInformation}>
+                        {rover.information}
+                    </p>
+                </VerticalTimelineElement>
+
+
+                <VerticalTimelineElement
+                    className="vertical-timeline-element--education"
+                    date="April 2013"
+                    contentStyle={{ background: '#631111', color: '#fff' }}
+                    contentArrowStyle={{ borderRight: '7px solid  #631111' }}
+                    iconStyle={{ background: '#631111', color: '#631111' }}
+                >
+                    <h3 className={styles.TimeLineHeading}>{rover.name}</h3>
+                    <img className={styles.RoverImage} src={rover.imageUrl} alt="rover" />
+                    <p className={styles.RoverInformation}>
+                        {rover.information}
+                    </p>
```

Comment:
so at the moment it just repeats the same data multiple times.
When its used for real what data are we expecting it to display?